### PR TITLE
Updates to convergence reporting module, sample convergence report

### PR DESCRIPTION
**Pull request by vivekyadav26**

### CODE REVIEW

**Comment by xyzjayne** [comment]
@i-am-sijia This PR includes a few updates on the convergence reporting modules.

- Interaction with income-segmented skims
- Using the "tabulate" package for converting dataframes into markdown
- Sample convergence report

File: examples/highway_convergence_report.txt [hunk 1]
```diff
@@ -0,0 +1,111 @@
+
```
Comment (i-am-sijia) [inline examples/highway_convergence_report.txt:1]: @vivekyadav26 why tracking this file?

File: tm2py/controller.py [hunk 2]
```diff
@@ -310,7 +310,8 @@ def _queue_components(self, run_components: Collection[str] = None):
             _final_components = [c for c in _final_components if c in run_components]
         
             #printing run components by iteration type (initial, global, final) in order
-            print('Initial components', _initial_components)
+            if self.config.run.start_iteration == 0:
```
Comment (i-am-sijia) [inline tm2py/controller.py]: I'm not in favor of adding this condition. @vivekyadav26

**Review by i-am-sijia — CHANGES_REQUESTED** [review]
@vivekyadav26 Please take a look and respond to my comments.

File: examples/highway_convergence_report.txt (hunk 1 above)
Comment (vivekyadav26) [reply examples/highway_convergence_report.txt:1]: We don't have to track this file.

File: tm2py/controller.py (hunk 2 above)
Comment (vivekyadav26) [reply tm2py/controller.py]: Sorry, I am missing the background with this change. Jayne made these changes before she left. Can you suggest what is acceptable here ?

Comment (i-am-sijia) [reply tm2py/controller.py]: I suggest reverting this change.

File: examples/highway_convergence_report.txt (hunk 1 above)
Comment (i-am-sijia) [reply examples/highway_convergence_report.txt:1]: Will you please add it to the gitignore?

Comment (vivekyadav26) [reply examples/highway_convergence_report.txt:1]: Added to gitignore.

File: tm2py/controller.py (hunk 2 above)
Comment (vivekyadav26) [reply tm2py/controller.py]: reverted the changes.

**Pull request merged**